rgd/geodata/api/search.py

```diff
@@ -420,8 +421,12 @@ def extent_summary(found, has_created=False):
     if found and found.count():
         if has_created:
             summary = found.aggregate(
-                acquisition__min=Min(Coalesce('acquisition_date', 'created')),
-                acquisition__max=Max(Coalesce('acquisition_date', 'created')),
+                acquisition__min=Min(
+                    Coalesce('acquisition_date', 'created'), output_field=DateTimeField()
+                ),
+                acquisition__max=Max(
+                    Coalesce('acquisition_date', 'created'), output_field=DateTimeField()
+                ),
```

Comment:
I have a brilliantly worded comment in [`SpatialEntryFilter.filter_acquired`](https://github.com/ResonantGeoData/ResonantGeoData/blob/timestamped-model/rgd/geodata/filters.py#L165-L166) regarding this bit:

> There is a special where filtering by acquisition, created is also filtered for `RasterEntry`.

The clarity is haunting. I think I remember noticing that `has_created` is only passed for rasters and replicated that logic over on the filter.

What's actually happening? What is the "special" for `RasterEntry`?

Reply:
If https://github.com/ResonantGeoData/ResonantGeoData/pull/356 gets merged we can drop this.